### PR TITLE
Rails project base and database creation

### LINKED ISSUE #8: Rails project base and database creation
# REQUIREMENTS
It is required to create the base of the project in Rails and create the database that was already in the diagram.

# DESIGN

### PR DESCRIPTION
**Pull request by diegoOG09**
# Description
- Creation of the project in Rails
- Creation of the database using the Rails Console
- Creation of foreign keys

  ## Screenshots

### CODE REVIEW

File: time-clock/config/database.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: db/test.sqlite3
+
+production:
+  <<: *default
+  database: db/production.sqlite3
```
Comment (YilKanda) [inline time-clock/config/database.yml:25]: This file should not be uploaded to GitHub, instead you can put an example file:

database.yml.example

With the recommended settings.

**Review by YilKanda — APPROVED** [review]

File: time-clock/db/migrate/20211222000109_create_employees.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    create_table :employees do |t|
+      t.string :name
+      t.string :position
+      t.integer :branch_id
+      t.integer :private_number
+      t.string :email
+      t.integer :check_in_out_id
+      t.boolean :active
+      #t.branch :belongs_to
+      #t.check_in_out :belongs_to
```
Comment (EduardoGHdez) [inline time-clock/db/migrate/20211222000109_create_employees.rb]: Pls, remove these comments if aren't necessary

File: time-clock/db/migrate/20211222000109_create_employees.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class CreateEmployees < ActiveRecord::Migration[6.1]
+  def change
+    create_table :employees do |t|
+      t.string :name
+      t.string :position
+      t.integer :branch_id
```
Comment (EduardoGHdez) [inline time-clock/db/migrate/20211222000109_create_employees.rb]: I think the `belongs_to` "branch" below statements covers the use case for this column, right?

File: time-clock/db/migrate/20211222000109_create_employees.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      t.integer :private_number
+      t.string :email
+      t.integer :check_in_out_id
```
Comment (EduardoGHdez) [inline time-clock/db/migrate/20211222000109_create_employees.rb]: Same as above

File: time-clock/db/migrate/20211222000227_create_admins.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+class CreateAdmins < ActiveRecord::Migration[6.1]
+  def change
+    create_table :admins do |t|
+      t.string :user
+      t.string :password
```
Comment (EduardoGHdez) [inline time-clock/db/migrate/20211222000227_create_admins.rb]: Password should not be saved as plain text, pls consider using something like "Devise" for the authentication
It is a classic gem for getting this feature to work without a lot of effort and it has battle tested code. It has probed to be secure a good start point

**Review by EduardoGHdez — COMMENTED** [review]
Looks good overall.

Let's discuss tomorrow at sync to give you some feedback related to my comments

File: time-clock/config/database.yml (hunk 1 above)
Comment (YilKanda) [inline time-clock/config/database.yml:25]: Add this file to the .gitignore

**Review by EduardoGHdez — APPROVED** [review]
LGTM

**Pull request merged**